### ISSUE TITLE
Problems after updating LuckPerms

### ISSUE THREAD

**Issue by DarkFort** [sgtcaze/NametagEdit#316]
![image](https://user-images.githubusercontent.com/25278617/47158437-541f7100-d2f4-11e8-90f0-60eda0cdc150.png)
![image](https://user-images.githubusercontent.com/25278617/47158454-5da8d900-d2f4-11e8-87bd-de5ac42ee3c3.png)
After updating the LuckPerms permissions plugin, nickname in the tab becomes white. This happens very often.

![image](https://user-images.githubusercontent.com/25278617/47158531-8d57e100-d2f4-11e8-8321-4b8f6a5ddb1a.png)
![image](https://user-images.githubusercontent.com/25278617/47158543-9052d180-d2f4-11e8-8a42-729b695099a6.png)

**Comment by DarkFort** [comment]
After the update, the issue was not resolved

**Comment by MrCrazyKill** [comment]
I've same issue.
`This server is running CraftBukkit version git-Spigot-db6de12-18fbb24 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)`
NametagEdit version 4.4.0
LuckPerms version 4.3.11

**Comment by JustOneMoreBlock** [comment]
Are you still having this issue?

**Comment by DarkFort** [comment]
Yes, there is a problem. I have to update the tab each time with the /nte reload command.

**Comment by Germanode** [comment]
Same issue here

**Comment by BestMark** [comment]
Same problem

**Comment by ThatsATim** [comment]
I have the same issue too
